### PR TITLE
[T3A2][F11-A4] Cara Leong Su-Yi

### PR DESCRIPTION
**Pull request by craaaa**

### CODE REVIEW

**Comment by craaaa** [comment]
Ready for review

**Comment by craaaa** [comment]
Ready for review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -33,5 +33,12 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * @return the feedbackToUser
+     */
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: consider removing useless header comments inserted by auto-gen. 
Accessors usually don't need header comments as their names are self-explanatory.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 2]
```diff
@@ -33,5 +33,12 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
...
+    public String getFeedbackToUser() {
+        return feedbackToUser;
+    }
+
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: you may have missed to change TextUi.java that accesses this variable directly.

**Comment by cmkumar87** [comment]
@craaaa Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Pull request closed without merging**